ci/download_dependencies.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+set -euxo pipefail
+
+source "${WORKDIR}/ci/archs.sh"
+
+temp_dir="${WORKDIR}/bin/deps/releases"
+lib_root="${WORKDIR}/bin/deps/lib"
+
+libtelio_version="v5.0.0-rc2"
+libtelio_url="${LIBTELIO_RELEASES_URL}/${libtelio_version}/downloads/linux.zip"
+libtelio_zipfile="${temp_dir}/libtelio-${libtelio_version}.zip"
+libtelio_dst="${temp_dir}/libtelio-${libtelio_version}"
+
+libdrop_version="v7.0.0"
+libdrop_url="${LIBDROP_RELEASES_URL}/${libdrop_version}/downloads/linux.zip"
+libdrop_zipfile="${temp_dir}/libdrop-${libdrop_version}.zip"
+libdrop_dst="${temp_dir}/libdrop-${libdrop_version}"
+
+libmoose_nordvpnapp_version="v11.1.0-nordVpnApp"
+libmoose_nordvpnapp_url="${LIBMOOSE_RELEASES_URL}/${libmoose_nordvpnapp_version}/downloads/linux.zip"
+libmoose_nordvpnapp_zipfile="${temp_dir}/libmoose-nordvpnapp-${libmoose_nordvpnapp_version}.zip"
+libmoose_nordvpnapp_dst="${temp_dir}/libmoose-nordvpnapp-${libmoose_nordvpnapp_version}"
+
+libmoose_worker_version="v10.0.4-worker"
+libmoose_worker_url="${LIBMOOSE_RELEASES_URL}/${libmoose_worker_version}/downloads/linux.zip"
+libmoose_worker_zipfile="${temp_dir}/libmoose-worker-${libmoose_worker_version}.zip"
+libmoose_worker_dst="${temp_dir}/libmoose-worker-${libmoose_worker_version}"
+
+mkdir -p "${temp_dir}"
+
+function fetch_gitlab_release() {
+  if [[ $# -ne 2 ]]; then
+    echo "Two parameters are required for fetch_gitlab_release function."
```

Comment:
[minor] This probably won't come up, as this script will not be called directly by people, but maybe just to be safe we could print how many parameters were provided and what parameters are required? Would help with debugging if something ever breaks here.

Reply:
Oh, that's a good idea :+1:, I'll change it